src/components/BuildBaronAndAnnotations/AnnotationTickets.tsx

```diff
@@ -1,22 +1,72 @@
-import React from "react";
+import React, { useState } from "react";
+import styled from "@emotion/styled";
+import { Tooltip } from "antd";
+import { ConditionalWrapper } from "components/ConditionalWrapper";
+import { PlusButton } from "components/Spawn";
 import { IssueLink } from "gql/generated/types";
+import { AddIssueModal } from "./AddIssueModal";
 import { AnnotationTicketsTable } from "./AnnotationTicketsTable";
 import { TicketsTitle, TitleAndButtons } from "./BBComponents";
 
 interface Props {
+  taskId: string;
+  execution: number;
   tickets: IssueLink[];
-  title: string;
+  isIssue: boolean;
+  userCanModify: boolean;
 }
 
-export const AnnotationTickets: React.FC<Props> = ({ tickets, title }) => (
-  <>
-    {tickets?.length && (
+export const AnnotationTickets: React.FC<Props> = ({
+  tickets,
+  taskId,
+  execution,
+  isIssue,
+  userCanModify,
+}) => {
+  const title = isIssue ? "Issues" : "SuspectedIssues";
+  const buttonText = isIssue ? "Add Issue" : "Add SuspectedIssue";
```

Comment:
should there be a space in SuspectedIssue

Reply:
fixed